projects/cds/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@spartacus/cds",
+  "version": "0.1.0-prealpha.0",
+  "description": "Context Driven Service library for Spartacus",
+  "keywords": [
+    "spartacus",
+    "framework",
+    "storefront",
+    "merchandising",
+    "segmentation",
+    "context-driven services",
+    "cds"
+  ],
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": "https://github.com/SAP/cloud-commerce-spartacus-storefront",
+  "peerDependencies": {
+    "@angular/common": "~8.0.0",
```

Comment:
Please align the versions with the ones in the main `package.json`

Reply:
Updated